### PR TITLE
Add spud-convert tool

### PR DESCRIPTION
**Pull request by aredridel**
A simple interface to the `spud.convert` call

### CODE REVIEW

File: bin/spud-convert [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+
+var spud = require('../index.js');
```
Comment (jasisk) [inline bin/spud-convert]: `var spud = require('../');`

Comment (aredridel) [inline bin/spud-convert]: Fixed!

**Comment by totherik** [comment]
Any usage details or documentation that should be added?

**Comment by aredridel** [comment]
Absolutely. Still thinking about direction and ultimate utility of it.

**Pull request closed without merging**